test/plugin/test_out_google_cloud.rb

```diff
@@ -70,6 +70,70 @@ def test_server_error
     assert_equal 1, exception_count
   end
 
+  def test_prometheus_requests_count
+    setup_gce_metadata_stubs
+    [
+      # Single successful request.
+      [200, 1, 1, 0],
+      # Several successful requests.
+      [200, 2, 2, 0],
+      # Single failed request.
+      [401, 1, 0, 1]
+    ].each do |code, emit_count, successful_count, failed_count|
+      setup_prometheus
+      stub_request(:post, WRITE_LOG_ENTRIES_URI)
+        .to_return(status: code, body: 'Some Message')
+      (1..emit_count).each do |i|
+        d = create_driver(PROMETHEUS_ENABLE_CONFIG)
+        d.emit('message' => log_entry(i.to_s))
+        # rubocop:disable Lint/HandleExceptions
+        begin
+          d.run
+        rescue Google::Apis::AuthorizationError
+        rescue Google::Apis::ServerError
+        end
+        # rubocop:enable Lint/HandleExceptions
+      end
+      assert_prometheus_metric_value(:stackdriver_successful_requests_count,
+                                     successful_count, grpc: false)
+      assert_prometheus_metric_value(:stackdriver_failed_requests_count,
+                                     failed_count, grpc: false, code: code)
+    end
+  end
+
+  def test_prometheus_ingested_entries
```

Comment:
Tbh -- b/c rubocop doesn't allow more than 5 values in the parameters list :)

Ack, I'll change it to one method